index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Register</title>
+  </head>
+  <link href="./CSS/style.css" rel="stylesheet" />
+  <script>
+
+  </script>
+  <body>
+    <div class="header"><header>Assignment 1</header></div>
+    <h1>Register</h1>
+    <form>
+      <div class="values">
+        <div class="fname">
+          <label for="first"> First Name:</label>
+          <input type="text" name="first" id="first" class="input" />
+        </div>
+       <div class="lname">
+         <label for="last"> Last Name:</label
+         ><input type="text" name="last" id="last" class="input" />
+         </div>
+        
+        <div class="email"><label for="email">Email:</label>
+          <input type="email" name="email" id="email" class="input" />
+        </div>
+        <div class="gender"><label for="gender">Gender:</label>
+          <input type="radio" id="male" name="gender" valu="male" />
+          <label for="male">Male</label>
+          <input type="radio" id="female" name="gender" valu="female" />
+          <label for="female">Female</label>
+          <input type="radio" id="other" name="gender" valu="other" /><label
+            for="other">Other</label>
+        </div>
+        <div class="clr"> <label for="favourite" class="input">Favourite Color:</label
+        ><select id="favclr">
+          <option value="" selected></option>
+          <option value="yellow">Yellow</option>
+          <option value="red">Red</option>
+          <option value="green">Green</option>
+          <option value="purple">Purple</option>
+        </select></div>
+
+        <div class="emp">
+          <label>Employeed</label>
+          <input type="checkbox" id="check" class="input" />
+        </div>
+        <div class="notes">
+          <label for="notes">Notes:</label>
+          <textarea id="notes" name="notes" rows="2" cols="8" class="input"></textarea>
+        </div>
+      
+      <input type="submit" id="submit" />
```

Comment:
add function like this to get form values
`function onSave(event){
event.preventDefault();
const x = {fname: document.getElementById("fname").value, lname: document.getElementById("lname").value};

console.log(x);
}`